test/mitmproxy/addons/test_view.py

```diff
@@ -138,6 +138,10 @@ def tdump(path, flows):
         for i in flows:
             w.add(i)
 
+def twrite_invalid_data(path):
+    invalid_data = b"invalidflows"
+    with open(path, "wb") as f:
+        f.write(invalid_data)
```

Comment:
Minor style suggestion: Inline this function. It's not going to be used somewhere else and is a two-liner if you just do `f.write(b"invalid")` :)